tables/sw-ke-g1-2.ctb

```diff
@@ -0,0 +1,105 @@
+# liblouis: Swahili (Kenya) (Grade 1.2)
+#
+#-index-name: Swahili (Kenya) Grade 1.2
+#-display-name: Swahili (Kenya) Grade 1.2
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#-updated: 2022
+#
+#+language:sw
+#+region:ke
+#+type:literary
+#+grade:1.2
+#
+# Copyright (C) 2022 by Compass Braille http://www.compassbraille.org
+#
+# This table was created using information contained in a document entitled
+# "Kenya Swahili Braille Code" which was derived from the Kenya Institute of
+# Curriculum Development's Swahili Braille Manual: “Mwongozo wa Breli Ya
+# Kiswahili”.
+#
+# The Braille code is graded into 6 grades: 1, 1.2, 1.3, 1.4, 1.5 and 2. This is
+# in line with the Kenya Institute of Curriculum Development’s Curriculum
+# Designs for Kiswahili Braille. 
+#
+# The Braille code in the document was adopted at a Stakeholders Workshop held
+# at the Swiss Lenana Mount Hotel on 25th August 2022.
+#
+# List of participating stakeholders:
+# 1. Ministry of Education – Directorate of Special Needs Education
+# 2. Ministry of Education – Directorate of Quality Assurance and Standards 
+# 3. Kenya Institute of Curriculum Development 
+# 4. Kenya Institute of Special Education
+# 5. Kenya Institute for the Blind
+# 6. Kenya Union of the Blind
+# 7. Kilimanjaro Blind Trust Africa
+# 8. Thika Primary School for the Blind 
+# 9. African Braille Centre
+# 10. Inclusion Resources for Africa
+# 11. International Council for the Education of People with Visual Impairment
+#     – Africa 
+#
+# Recommendations from Stakeholders
+# 1. The Code to be known as the “Kenya Swahili Braille Code” or
+#    “Swahili (Kenya)”
+# 2. Any other existing Swahili codes associated with Kenya to be replaced with
+#    the proposed code. 
+# 3. Any other Swahili Braille codes to remain but to be distinguished from the
+#    proposed code.
+# 4. The Kenya Swahili Braille code to consist of 6 Grades as categorized.
```

Comment:
I'm not sure whether this whole text from the "Kenya Swahili Braille Code" document including the list of stakeholders needs to be present here. If we link to the braille-specs page (https://github.com/liblouis/braille-specs/pull/20), we should be able to drop at least a part of the text. Do you agree?

Reply:
I've removed text requested and added a link. I guess I can't put the final link till Swahili braille-specs PR is merged?

Reply:
Thanks. Yes, you can already put the final link if you want. This PR will then just be blocked on the other.

Reply:
So:
http://liblouis.org/braille-specs/swahili/
?

Reply:
Yes